README.md

```diff
@@ -28,19 +28,20 @@ New here or looking to install? Check [our website](https://randovania.github.io
 
 <!-- Begin WELCOME -->
 
-Here you will be able to randomize many aspects of either game, while still being ensured it's possible to
-finish without any trick or glitch! What can be randomized?
+Randovania can randomize many aspects of its supported games, all while still ensuring they're completable 
+without any unintended tricks or glitches. Its features include:
 
-* Randomize what can be found in each pickup location, including major upgrades, expansions, keys and artifacts.
+* Randomizing what can be found in each pickup location. Weapons, keys, and more can end up in
+  completely new places.
 
-* Play with multiple people, via multiworld sessions. Your pickups will be shuffled among the games of
-everyone involved, no matter what game they're playing!
+* Use multiworld sessions to shuffle items between multiple separate games, alone or with friends.
```

Comment:
Hmm, I feel like it would still be worth to have a very brief explanation on how MW works.

Reply:
Agreed - I mainly wanted to make it clear multiworld =/= multiplayer. What information do you think is missing? I'm happy to make this slightly longer for clarity.

Reply:
The `Your pickups will be shuffled among the games of everyone involved` part explained fairly quick n simple on how it works. Altho I it'd be better to say "items" than "pickups"

Reply:
I might be having a brain moment; doesn't "shuffle items between multiple separate games" establish the same information?

Reply:
It does, *I* was having a brain moment